core/src/test/scala/caliban/relay/ConnectionSpec.scala

```diff
@@ -184,7 +185,7 @@ object ConnectionSpec extends DefaultRunnableSpec {
             )
           )
         )
-      },
+      } @@ nonFlaky,
```

Comment:
Since there is no more validation involved, I guess we don't need this?

Reply:
Nah true, I just wanted to prove that it actually was nonFlaky :D I'll remove it!